modules/n1ql/pages/n1ql-language-reference/numericfun.adoc

```diff
@@ -4,121 +4,116 @@
 
 {description}
 
-ABS(expression)
+== ABS(expression)
 
 Returns absolute value of the number.
 
-ACOS(expression)
+== ACOS(expression)
 
 Returns arccosine in radians.
 
-ASIN(expression)
+== ASIN(expression)
 
 Returns arcsine in radians.
 
-ATAN(expression)
+== ATAN(expression)
 
 Returns arctangent in radians.
 
-ATAN2(expression1, expression2)
+== ATAN2(expression1, expression2)
 
 Returns arctangent of expression2/expression1.
 
-CEIL(expression)
+== CEIL(expression)
 
 Returns smallest integer not less than the number.
 
-COS(expression)
+== COS(expression)
 
 Returns cosine.
 
-DEGREES(expression)
+== DEGREES(expression)
 
 Returns radians to degrees.
 
-E()
+== E()
 
 Base of natural logarithms.
 
-EXP(expression)
+== EXP(expression)
 
 Returns e^expression^.
 
-LN(expression)
+== LN(expression)
 
 Returns log base e.
 
-LOG(expression)
+== LOG(expression)
 
 Returns log base 10.
 
-FLOOR(expression)
+== FLOOR(expression)
 
 Largest integer not greater than the number.
 
-PI()
+== PI()
 
 Returns PI.
 
-POWER(expression1, expression2):
+== POWER(expression1, expression2):
 
 Returns expression1^expression2^.
 
-RADIANS(expression)
+== RADIANS(expression)
 
 Returns degrees to radians.
 
-RANDOM([ expression ])
+== RANDOM([ expression ])
 
 Returns pseudo-random number with optional seed.
 
-ROUND(expression [, digits ])
+== ROUND(expression [, digits ])
 
 Rounds the value to the given number of integer digits to the right of the decimal point (left if digits is negative).
 Digits is 0 if not given.
 
-SIGN(expression)
+== SIGN(expression)
 
 Valid values:  -1, 0, or 1 for negative, zero, or positive numbers respectively.
 
-SIN(expression)
+== SIN(expression)
 
 Returns sine.
 
-SQRT(expression)
+== SQRT(expression)
 
 Returns square root.
 
-TAN(expression)
+== TAN(expression)
 
 Returns tangent.
 
-TRUNC(expression [, digits ])
+== TRUNC(expression [, digits ])
 
 Truncates the number to the given number of integer digits to the right of the decimal point (left if digits is negative).
 Digits is 0 if not given.
 
-Example
+== Example
+====
 
+include::ROOT:partial$query-context.adoc[tag=example]
+
+.Query
+
+[source, sqlpp]
 ----
-SELECT
-    AVG(reviews.rating) / 5 AS normalizedRating,
-    ROUND((avg(reviews.rating) / 5), 2) AS roundedRating,
-    TRUNC((avg(reviews.rating) / 5), 3) AS truncRating
-    FROM reviews AS reviews
-    WHERE reviews.customerId = "customer62"
+include::example$n1ql-language-reference/numeric-func.n1ql[]
```

Comment:
This directory is for DDL examples, can you put this example in the `functions` directory?